packages/yoga/src/hooks/useKeyPress.jsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react';
+
+export const useKeyPress = targetKey => {
+  const [keyPressed, setKeyPressed] = useState(false);
+
+  useEffect(() => {
+    function downHandler({ key }) {
+      if (key === targetKey) {
+        setKeyPressed(true);
+      }
+    }
+    function upHandler({ key }) {
+      if (key === targetKey) {
+        setKeyPressed(false);
+      }
+    }
+    window.addEventListener('keydown', downHandler);
+    window.addEventListener('keyup', upHandler);
+
+    return () => {
+      window.removeEventListener('keydown');
```

Comment:
Nice contribution, @guilhermecardoso-gympass 

removeEventListener should receive two params:
```suggestion
       window.removeEventListener('keydown', downHandler);
      window.removeEventListener('keyup', upHandler);
```

Where is this hook being used?

Reply:
Done!